csharp/src/Ice/LocatorInfo.cs

```diff
@@ -271,7 +271,9 @@ async Task<IReadOnlyList<Endpoint>> PerformGetAdapterEndpointsAsync(Reference re
                 try
                 {
                     // TODO: Fix FindAdapterById to return non-null proxy
-                    IObjectPrx? proxy = await Locator.FindAdapterByIdAsync(reference.AdapterId).ConfigureAwait(false);
+                    IObjectPrx? proxy = await Locator.FindAdapterByIdAsync(
+                        reference.AdapterId,
+                        cancel: CancellationToken.None).ConfigureAwait(false);
```

Comment:
Why is this explicit cancel parameter necessary?

Reply:
the analyzer detects methods that has a cancel parameter and require to explicit pass one, I think this is to avoid forgetting to forward the parameter. 

https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2016?view=vs-2019

Keep this rule enable will remind us to forward the cancel parameters, and explicit pass default / None if the param shouldn't be forward or there isn't a param to forward.